tests/browser/.env.example

```diff
@@ -1,4 +1,4 @@
 USE_LOCAL_API=true
 WEBSITE_HOST=http://localhost:5000
 RELYING_PARTY_URL=http://example.net
-ENV="dev"
+ENVIRONMENT="dev"
```

Comment:
Do we use `ENV` or `ENVIRONMENT` in other repos?
It would be good if this was consistent